.github/policies/flaggedPackages.yml

```diff
@@ -123,7 +134,7 @@ configuration:
 
                 | aloneguid (Ivan G) Browser Tamer | Publisher has requested removal | #115266 (comment) |
 
-
+                | FFXIV Teamcraft | Publisher has requested removal | #115369 |
```

Comment:
You need the extra newline here or the message formatting will be weird

Reply:
I have added the newline in the file. Not sure why GitHub diffs are not showing this properly
![image](https://github.com/microsoft/winget-pkgs/assets/88161975/1f8d69de-8f9f-4b31-be1a-5f998b2c961e)

![image](https://github.com/microsoft/winget-pkgs/assets/88161975/87f6326d-9811-4fd4-ac57-8b0279986e64)

Reply:
Weird